WIP-CHANGELOG.md

```diff
@@ -31,7 +31,7 @@
 - Fixed harbor network policies
 
 ### Added
-
+- Option to configure alerts for growing indices in OpenSearch
```

Comment:
Maybe make a note in the release notes on that these new option should or might be tweaked to better suit the environment